src/pai_rag/app/web/ui_constants.py

```diff
@@ -32,16 +32,52 @@
         }
         """
 
-DEFAULT_EMBED_SIZE = 1536
+DEFAULT_EMBED_SIZE = 1024
+
+DEFAULT_HF_EMBED_MODEL = "bge-large-zh-v1.5"
+
+
+EMBEDDING_MODEL_DEPRECATED = [
+    "bge-small-zh-v1.5",
+    "SGPT-125M-weightedmean-nli-bitfit",
+    "text2vec-large-chinese",
+    "paraphrase-multilingual-MiniLM-L12-v2",
+]
+
+EMBEDDING_MODEL_LIST = [
```

Comment:
Maybe we can add  some comments of links to the embedding models

Reply:
Done